rpc/README.md

```diff
@@ -60,6 +60,18 @@ Retrieves block header by given block number.
 
 * `block_header`: `BlockHeader` – block header.
 
+### GetNotesById
+
+Returns a list of Note's matching a certain list of NoteId's.
+
+**Parameters**
+
+* `note_ids`: `NoteId` - ID of the Note that we want to query.
+
+**Returns**
+
+* `notes`: `Note` - List of notes matching the list of NoteId's.
+
```

Comment:
nit: I think this is missing the changes done related to inputs and outputs being lists.

Reply:
fixed by merging next.

Reply:
Seems like something is still off here. This content is different from what's in `next` (see [here](https://github.com/0xPolygonMiden/miden-base/blob/next/miden-tx/src/host/account_delta_tracker.rs)). Also, it shouldn't appear as new changes.